.github/workflows/ci.yml

```
@@ -69,6 +69,10 @@ jobs:
       - run: yarn test:unit --forceExit --silent
         env:
           NODE_OPTIONS: --max_old_space_size=20480
+      - name: Upload coverage reports to Codecov
```

It looks like this step wouldn't run in the case where coverage was too low. The report would be helpful then in showing where to increase coverage. You could ensure that this always runs by doing something like this:

```suggestion
      - name: Upload coverage reports to Codecov
        if: ${{ always() }}
```

Not sure how that would act if it failed during setup though, or if no coverage results exist. It might handle it gracefully.